components/logins/src/engine.rs

```diff
@@ -204,27 +202,23 @@ mod test {
         let a_id = engine.add(a.clone()).expect("added a");
         let b_id = engine.add(b.clone()).expect("added b");
 
-        assert_eq!(a_id, a.guid);
-
-        assert_ne!(b_id, b.guid, "Should generate guid when none provided");
-
+         //Get item a's ID, make sure no errors getting a and for a's ID to exist - ben and ryan
```

Comment:
General nit: should have a space after `//`, for example ``// Get...`` vs. ``//Get``.